package.json

```diff
@@ -466,7 +465,7 @@
     "node-fetch": "2.6.7",
     "browserslist": "4.16.6",
     "ws": "8.8.0",
-    "css-what": "5.0.1",
+    "css-what": "6.1.0",
```

Comment:
I'm worried this might break something. I was dealing with this the other day, and there are a lot of packages depending on different versions of css-what, and I think 6 is not backwards compatible enough with 5. :/